### ISSUE TITLE
Determine if a site is in the sea

### ISSUE THREAD

**Issue by joeroe** [xronos-ch/xronos.rails#326]
We can detect whether to use the northern or southern hemisphere curve easily enough (if we have a latitude coordinate), but how can we know whether to use MarineCal?

**Comment by MartinHinz** [comment]
### Summary of Options for Checking if a Location is in the Sea

#### 1. **Using External APIs**
- **MarineRegions.org**: Use their Gazetteer Web Services to determine if coordinates are in a marine area. Note that as of the last check, this service may return a 404 error.
- **GeoNames API**: Utilize the `oceanJSON` endpoint to check if coordinates are in the ocean. Requires signing up for a free GeoNames account.
- **OpenStreetMap Nominatim API**: Reverse geocode the coordinates to check if the location is described as water or ocean.

#### 2. **Using Geospatial Data Locally**
- **PostGIS with PostgreSQL**:
  - Install PostgreSQL and PostGIS.
  - Import shapefiles for land and ocean polygons from sources like Natural Earth.
  - Perform spatial queries to check if the coordinates are within ocean polygons.

- **Without PostGIS**:
  - **GeoJSON Files with `rgeo`**:
    - Download GeoJSON files for land and ocean polygons.
    - Use the `rgeo` gem to load the GeoJSON data and perform point-in-polygon checks.

  - **Shapefiles with `ruby-shapefile` and `rgeo`**:
    - Download shapefiles for land and ocean polygons.
    - Use the `ruby-shapefile` gem to read the shapefiles.
    - Use the `rgeo` gem to perform point-in-polygon checks.

**Comment by joeroe** [comment]
I don't know what I was thinking with the former title: the question is not whether the site is in the sea, but where the sample came from. MarineCal is for *samples* from a marine environment and most of these show up on terrestrial sites. Conversely, samples from submarine sites (if people do radiocarbon dating of them??) are likely terrestrial in origin and should use IntCal/SHCal.

**Comment by MartinHinz** [comment]
In that case, to my knowledge, there is no way of telling this directly from the data that are usually published with the date itself. The only logic, that I can think of, would be, if we have a Delta C13 value, and that value indicates a marine influence, then we could suggest to use it to use the marine curves. Do you have any other ideas?

**Comment by joeroe** [comment]
Yeah I agree, it's not feasible to do this reliably. Defaulting to IntCal or SHCal, and giving the user the option to switch to MarineCal, seems fine.
